tests/manage/mcg/test_namespace.py

```diff
@@ -0,0 +1,263 @@
+import logging
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1
+)
+from tests.helpers import create_unique_resource_name
+from ocs_ci.ocs.bucket_utils import (
+    sync_object_directory,
+    verify_s3_object_integrity
+)
+logger = logging.getLogger(__name__)
+
+
+class TestNamespace:
+    """
+    Test creation of a namespace resource
+    """
+    aws_endpoint = 'https://s3.amazonaws.com'
+    ns_resource = 'ns_resource'
+    ns_bucket = 'ns-bucket'
+    aws_connection = 'aws_connection'
+    region = 'us-west-1'
+    result_dir = '/result'
+    original_dir = '/original'
+
+    @tier1
```

Comment:
we need to add Polarion IDs. @fbalak FYI

Reply:
Polarion IDs provided

Reply:
added the provided Polarion IDs to code